mvnw

```diff
@@ -0,0 +1,316 @@
+#!/bin/sh
```

Comment:
Yeah this file come from maven wrapper, helps to run maven commands without having maven installed in the machine. Will try to use this in docker file as well to avoid pulling in maven's image.